nextcord/message.py

```diff
@@ -517,7 +518,7 @@ class Message(Hashable):
         A :class:`Member` that sent the message. If :attr:`channel` is a
         private channel or the user has the left the guild, then it is a :class:`User` instead.
     content: :class:`str`
-        The actual contents of the message.
+        The actual contents of the message. Return an empty string if ``Intents.messages_content`` is not enabled.
```

Comment:
No need to mention this in my opinion. Bots can still get message content when it is disabled for some messages, such as ones the bot sent, ones where the bot is mentioned, and ones sent by DMs.

```suggestion
        The actual contents of the message.
```